diff --git a/pkg/builder/step_mount_extra.go b/pkg/builder/step_mount_extra.go
index bcc90f0..6dcc90a 100644
--- a/pkg/builder/step_mount_extra.go
+++ b/pkg/builder/step_mount_extra.go
@@ -104,7 +104,7 @@ func (s *StepMountExtra) CleanupFunc(state multistep.StateBag) error {
 			}
 		}
 
-		unmountCommand, err := wrappedCommand(fmt.Sprintf("umount %s", path))
+		unmountCommand, err := wrappedCommand(fmt.Sprintf("umount -f -R %s", path))
 		if err != nil {
 			return fmt.Errorf("Error creating unmount command: %s", err)
 		}
